astrodb_utils/utils.py

```diff
@@ -68,21 +67,22 @@ class AstroDBError(Exception):
 #     yield
 #     sys.tracebacklimit = default_value  # revert changes
 
-# TODO:  Where should these live?
-REFERENCE_TABLES = [
-    "Publications",
-    "Telescopes",
-    "Instruments",
-    "Modes",
-    "PhotometryFilters",
-    "Versions",
-    "Parameters",
-]
-
 
-def load_astrodb(db_file, recreatedb=True, reference_tables=REFERENCE_TABLES):
+def load_astrodb(db_file, data_path="data/", recreatedb=True, reference_tables=None):
     # Utility function to load the database
 
+    if reference_tables is None:
+        logger.warning("No reference tables provided. Using default tables.")
+        REFERENCE_TABLES = [
```

Comment:
Usually all CAPS are global variables, we should keep it as `reference_tables`, but honestly we should just have this as the parameters in the function definition itself.